MyCryptoHQ/MyCrypto
# Address Book Labeling in add-existing-account

### Description
Implements address book labels.

### Changes
* Adds handling for default account labeling
* Makes UUID not tied to address

To Test:
1 ) Add multiple accounts using the same network. It should increment their names and add the labels to local cache.
2 ) Add multiple accounts using different networks. Should not increment names, but will add labels to local cache.
